Source/ASTextNode2.mm

```diff
@@ -617,6 +615,63 @@ - (id)_linkAttributeValueAtPoint:(CGPoint)point
   return nil;
 }
 
+- (BOOL)_locked_pointInsideAdditionalTruncationMessage:(CGPoint)point withLayout:(ASTextLayout *)layout
+{
+  // Check if the range is within the additional truncation range
+  BOOL inAdditionalTruncationMessage = NO;
+  
+  CTLineRef truncatedCTLine = layout.truncatedLine.CTLine;
+  if (truncatedCTLine != NULL && _additionalTruncationMessage != nil) {
+    CFIndex stringIndexForPosition = CTLineGetStringIndexForPosition(truncatedCTLine, point);
+    if (stringIndexForPosition != kCFNotFound) {
+      CFIndex truncatedCTLineGlyphCount = CTLineGetGlyphCount(truncatedCTLine);
+      
+      CTLineRef truncationTokenLine =
+      CTLineCreateWithAttributedString((CFAttributedStringRef)_truncationAttributedText);
```

Comment:
Bump this to the line above, and on the several lines below. No 100 char line limits in Texture, thank goodness!